app/faq/ru.md

```diff
@@ -30,9 +30,9 @@ Est autem, reprehenderit doloremque ipsa, quos dolores, quisquam accusantium ut
 
 Lorem ipsum dolor sit amet, consectetur adipisicing elit. Perferendis porro esse molestias. Ex veniam itaque, quia quidem assumenda vitae in, aperiam fugit harum, consequuntur tempora blanditiis excepturi, temporibus?
```

Comment:
This file is outdated. The content needs to be replaced with the content of en.md and then translated.